src/omero_zarr/util.py

```diff
@@ -112,3 +122,55 @@ def marshal_transformations(
         zooms["y"] = zooms["y"] * multiscales_zoom
 
     return transformations
+
+
+def get_minimum_image_ome_xml(images: List[ImageWrapper]) -> str:
```

Comment:
Might be worth considering `ome_model` or `ome_types` for this? cc: @seb

Reply:
Using ome-types in [553b35f](https://github.com/ome/omero-cli-zarr/pull/123/commits/553b35f498764274f8dc882fb54c7e0a30fb6737)